test/compliance_tool/test_compliance_check_aasx.py

```diff
@@ -132,7 +132,7 @@ def test_check_aasx_files_equivalence(self) -> None:
                       manager.format_step(4, verbose_level=1))
         self.assertEqual(Status.NOT_EXECUTED, manager.steps[5].status)
 
-    @unittest.skipIf(sys.version.startswith("3.6"), "The XML schema check fails for Python 3.6")
+    @unittest.skipIf((3, 6) <= sys.version_info < (3, 7), "The XML schema check fails for Python 3.6")
```

Comment:
The test will probably fail for all versions below 3.6 as well, so even while they are not officially supported, we should just check `sys.version_info < (3, 7)` IMO.